src/System.Console/src/System.Console.builds

```diff
@@ -9,23 +9,8 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Console.csproj">
-      <OSGroup>Unix</OSGroup>
-      <TargetGroup>netstandard1.3</TargetGroup>
-    </Project>
-    <Project Include="System.Console.csproj">
-      <OSGroup>Windows_NT</OSGroup>
-      <TargetGroup>netstandard1.3</TargetGroup>
-    </Project>
-    <Project Include="System.Console.csproj">
-      <TargetGroup>net46</TargetGroup>
-    </Project>
-    <Project Include="System.Console.csproj">
       <TargetGroup>net463</TargetGroup>
     </Project>
-    <Project Include="System.Console.csproj">
```

Comment:
https://www.nuget.org/packages/runtime.win.System.Console/4.0.0 does have runtimes\win\lib\netcore50\ asset. Was that the thing that you were looking for?